.ci/jenkins/Jenkinsfile.e2e.cluster

```diff
@@ -83,6 +110,7 @@ pipeline {
                                 make test-e2e
                             """
                         } catch (err) {
+                            kind.exportLogs("/tmp/kind-logs")
```

Comment:
here could be the variable used. If the var usage directly does not work, try setting env variable instead.